recipe/install-libxgboost.sh

```diff
@@ -30,5 +30,4 @@ mkdir -p ${LIBDIR} ${INCDIR}/xgboost ${BINDIR} || true
 cp ${SRC_DIR}/xgboost${EXEEXT} ${BINDIR}/
 cp ${SRC_DIR}/lib/${XGBOOSTDSO} ${SODIR}/
 cp -Rf ${SRC_DIR}/include/xgboost ${INCDIR}/
-cp -Rf ${SRC_DIR}/rabit/include/rabit ${INCDIR}/xgboost/
```

Comment:
So should we drop this line?

Reply:
Yes

Reply:
Am still seeing [`rabit` in 2.1.0]( https://github.com/dmlc/xgboost/tree/v2.1.0/rabit ). Is it not suppose to be there or does it just get removed later?

Reply:
Looks like it is just used in a couple of C++ source files:

```
% git grep 'rabit/internal/socket\.h'
src/collective/loop.cc:#include "rabit/internal/socket.h"      // for PollHelper
src/collective/socket.cc:#include "rabit/internal/socket.h"      // for PollHelper
src/collective/tracker.cc:#include "rabit/internal/socket.h"
```

Plus it is removed in `master`

So seems ok to drop even though there is still something left in there. Users shouldn't need it